### PR TITLE
feat: update AUP for better law compliance

### PR DESCRIPTION
**Pull request by nerrixDE**

### CODE REVIEW

File: Acceptable Usage Policy.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Please note we are *committed to resolving every single report*, please be patie
 
 Engaging in any of the following will result in **immediate account termination** as well as content removal. You may also be **blacklisted** from using our services indefinitely.
 
-- Sharing any content or imagery illegal within the European Union, we don't want you or us to get into any legal trouble, stay in line by not engaging in illegal activity content such as, but not limited to, sharing pornography depicting minors, sharing revenge pornography, selling or facilitating the usage of drugs, extortion or blackmail.
+- Sharing any content or imagery illegal within the European Union, we don't want you or us to get into any legal trouble, stay in line by not engaging in illegal activity content such as, but not limited to, sharing any pornography that is subject to legal age restrictions, depicting minors, sharing revenge pornography, selling or facilitating the usage of drugs, extortion or blackmail.
```
Comment (nearlySplat) [inline Acceptable Usage Policy.md:35]:
```suggestion
- Sharing any content or imagery illegal within the European Union, we don't want you or us to get into any legal trouble, stay in line by not engaging in illegal activity content such as, but not limited to, sharing any pornography that is subject to legal age restrictions, depicting minors sexually, sharing revenge pornography, selling or facilitating the usage of drugs, extortion or blackmail.
```

Comment (nerrixDE) [reply Acceptable Usage Policy.md:35]: thanks but the wording we choose is fine

Comment (Rexogamer) [reply Acceptable Usage Policy.md:35]: grammar:
```suggestion
- Sharing any content or imagery illegal within the European Union. We don't want you or us to get into any legal trouble, so stay in line by not engaging in illegal activity content such as, but not limited to: sharing any pornography that is subject to legal age restrictions, depicting minors, sharing revenge pornography, selling or facilitating the usage of drugs, extortion or blackmail.
```

**Comment by nerrixDE** [comment]
@insertish merge needed

**Comment by nerrixDE** [comment]
🚔 wiuwiu @insertish

**Pull request merged**